roles/kubernetes-apps/csi_driver/cinder/templates/cinder-csi-controllerplugin.yml.j2

```diff
@@ -35,6 +35,9 @@ spec:
           image: {{ cinder_csi_provisioner_image_repo }}:{{ cinder_csi_provisioner_image_tag }}
           args:
             - "--csi-address=$(ADDRESS)"
+{% if cinder_topology is defined and cinder_topology is sameas true %}
```

Comment:
`cinder_topology` is supposed to be a boolean, so why doing the `is sameas`? Wouldn't `and cinder_topology` be equivalent?

Reply:
Correct cinder_topology is expected to be a boolean. This is just How I write code. I pushed another commit that changes this to keep the code streamlined :)